src/narrativelog/create_tables.py

```diff
@@ -60,8 +60,14 @@ def create_message_table(metadata: sa.MetaData) -> sa.Table:
         sa.Column("date_invalidated", saty.DateTime(), nullable=True),
         sa.Column("parent_id", UUID(as_uuid=True), nullable=True),
         # Added 2022-07-19
+        # 'systems' field is deprecated and will be removed in v1.0.0.
```

Comment:
I don't know any foolproof way to do deprecation for an API unless the API is always accessed via a paired client. Pairing client and API would be important if there were lots of non-Rubin users using the API. I assume that is NOT the case (if my assumption is wrong, maybe we should talk about what I've done in NOIRLab).  But, it would be good to add an API endpiont that gives a semantic version.  API documentation could warn that when the major component of the version increments, there have backwards incompatible changes.  That would at least let API user know when an API change put their (API using) code in a dangerous place.

Reply:
Got it :+1: I'll work on adding a `/version` endpoint.